arm-usage/2015-06-01-preview/swagger/usage.json

```diff
@@ -0,0 +1,423 @@
+          },
+          "description": "A list of offer terms."
+        },
+        "Meters": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MeterInfo"
+          },
+          "description": "A list of meters."
+        }
+      }
+    },
+    "MeterInfo": {
+      "description": "Detailed information about the meter.",
+      "properties": {
+        "MeterId": {
+          "description": "The unique identifier of the resource.",
+          "type": "string",
+          "format": "uuid"
+        },
+        "MeterName": {
+          "description": "The name of the meter, within the given meter category",
+          "type": "string"
+        },
+        "MeterCategory": {
+          "description": "The category of the meter, e.g., 'Cloud services', 'Networking', etc..",
+          "type": "string"
+        },
+        "MeterSubCategory": {
+          "description": "The subcategory of the meter, e.g., 'A6 Cloud services', 'ExpressRoute (IXP)', etc..",
+          "type": "string"
+        },
+        "Unit": {
+          "description": "The unit in which the meter consumption is charged, e.g., 'Hours', 'GB', etc.",
+          "type": "string"
+        },
+        "MeterRates": {
+          "description": "The list of key/value pairs for the meter rates, in the format 'key':'value' where key = the meter quantity, and value = the corresponding price",
+          "type": "object",
+          "additionalProperties": {
+            "type": "number",
+            "format": "float"
+          }
+        },
+        "EffectiveDate": {
+          "description": "Indicates the date from which the meter rate or offer term is effective.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "IncludedQuantity": {
+          "description": "The resource quantity that is included in the offer at no cost. Consumption beyond this quantity will be charged.",
+          "type": "number",
+          "format": "float"
+        }
+      }
+    },
+    "OfferTermInfo": {
+      "description": "Describes the offer term.",
+      "properties": {
+        "Name": {
+          "description": "Name of the offer term. For example: 'Monetary Credit",
+          "type": "string"
+        },
+        "TieredDiscount": {
+          "description": "The list of key/value pairs for the tiered meter rates, in the format 'key':'value' where key = price, and value = the corresponding discount percentage. This field is used only by offer terms of type 'Monetary Commitment'.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "number",
+            "format": "decimal"
+          }
+        },
+        "Credit": {
+          "description": "The amount of credit provided under the terms of the given offer level. This field is used only by offer terms of type 'Monetary Commitment'.",
+          "type": "number",
+          "format": "decimal"
```

Comment:
@moinak3, @Matt-Westphal 
As per the description in the documentation https://msdn.microsoft.com/en-us/library/azure/mt219004.aspx

> Credit  -  The amount of credit provided under the terms of the given offer level. This field is used only by offer terms of **type “Monetary Commitment”.**

>  TieredDiscount - The list of key/value pairs for the tiered meter rates, in the format “key”:”value” where key = price, and value = the corresponding discount percentage. This field is used only by **offer terms of type “Monetary Commitment”.**

The model definition as I have now looks incorrect. It seems that are different types of offer terms that the user can get. Since, the REST API doc did not have a proper schema of the response, I was not able to deduce the info accurately. 

It would be great if you could review and provide more info about this.